src/System.Console/src/System/ConsolePal.WinRT.cs

```diff
@@ -82,19 +82,27 @@ public static bool TreatControlCAsInput
             set { throw new PlatformNotSupportedException(); }
         }
 
+        private const ConsoleColor UnknownColor = (ConsoleColor)(-1);
```

Comment:
With this const now being used by the majority of the implementations, maybe move it to bring internal on the Console class?